pootle/apps/pootle_fs/management/commands/fs_commands/add.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+from pootle_fs.management.commands import FSAPISubCommand
+
+
+class AddCommand(FSAPISubCommand):
+    help = "Add translations into Pootle from FS."
+    api_method = "add"
+
+    def add_arguments(self, parser):
+        super(AddCommand, self).add_arguments(parser)
+        parser.add_argument(
+            '--force',
+            action='store_true',
+            dest='force',
+            help=('Add translations that are conflicting with Pootle '
+                  'stores'))
```

Comment:
I see this proposed message might not be the best one: Mark to override conflicts with FS translations.